src/Fixer/ClassNotation/VisibilityRequiredFixer.php

```diff
@@ -176,6 +176,11 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
 
         $propertyTypeDeclarationKinds = [T_STRING, T_NS_SEPARATOR, CT::T_NULLABLE_TYPE, CT::T_ARRAY_TYPEHINT, CT::T_TYPE_ALTERNATION, CT::T_TYPE_INTERSECTION, CT::T_DISJUNCTIVE_NORMAL_FORM_TYPE_PARENTHESIS_OPEN, CT::T_DISJUNCTIVE_NORMAL_FORM_TYPE_PARENTHESIS_CLOSE];
 
+        if (\defined('T_PRIVATE_SET')) { // @TODO: drop condition when PHP 8.4+ is required
+            $propertyAsymetricType = [T_PRIVATE_SET, T_PROTECTED_SET, T_PUBLIC_SET];
+            array_push($propertyTypeDeclarationKinds, ...$propertyAsymetricType);
```

Comment:
```suggestion
            array_push($propertyTypeDeclarationKinds, T_PRIVATE_SET, T_PROTECTED_SET, T_PUBLIC_SET);
```
Can it be like this?

Reply:
I have just changed the variable name.

Reply:
The new name is ok ? Or should I fallback to the previous?

Reply:
The m is **missing**, now in the PR title too :)

_https://wiki.php.net/rfc/asymmetric-visibility-v2_